pkg/webhook/policy.go

```diff
@@ -156,7 +158,7 @@ func (h *validationHandler) Handle(ctx context.Context, req admission.Request) a
 	}()
 
 	// namespace is excluded from webhook using config
-	if h.skipExcludedNamespace(req.AdmissionRequest.Namespace) {
+	if h.skipExcludedNamespace(req.AdmissionRequest.Namespace) || (req.AdmissionRequest.Kind.Kind == namespaceStr && req.AdmissionRequest.Kind.Group == "" && h.skipExcludedNamespace(req.AdmissionRequest.Name)) {
```

Comment:
We should update process excluder to handle this edge case. Otherwise we'll need to update:

* this webhook
* mutation webhook
* audit process
* sync controller

IMO, best to keep this fix in process excluder so that this bug doesn't crop up again if/when we write new code.

Reply:
moved inside excluder